specification/dns/resource-manager/Microsoft.Network/stable/2018-05-01/dns.json

```diff
@@ -1021,6 +1072,43 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/getDnsResourceReference": {
+      "post": {
```

Comment:
Is there a reason this is a `POST` operation, looks more like a `GET` operation

Reply:
It requires a list of resource Ids in the body. We already reviewed all API changes with ARM team.